Palo Alto Networks/paloalto-ngfw/tests/test_timestamp_palo.json

```diff
@@ -0,0 +1,49 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Palo Alto Next-Generation Firewall",
+        "dialect_uuid": "903ec1b8-f206-4ba5-8563-db21da09cafd"
+      }
+    },
+    "message": ": 1,2023/08/01 04:03:24,026701002348,SYSTEM,general,2816,2023/08/01 04:03:24,,general,,0,0,general,informational,\"Request made to server \"\"app-registry-service.apps.paloaltonetworks.com\"\" is successful . \",7261972653022396272,0x8000000000000000,0,0,0,0,,fwwan-hdr,0,0,2023-08-01T04:03:24.705+02:00"
```

Comment:
Is the colon at the beginning of the message is expected?

Reply:
yes of course